robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -150,6 +149,12 @@
                 w = DQ.E*DQ.k;
             end
         end
+
+
+        function ret = get_supported_joint_types(~)
```

Comment:
Is there any reason for ignoring the arguments? (That is, for using ~)

Reply:
Hi @bvadorno, I modified the current signature of the method to


```
methods (Static, Access = protected) 
     function ret = get_supported_joint_types()
     % This method returns the supported joint types.
        ret = [DQ_JointType.REVOLUTE, DQ_JointType.PRISMATIC];
     end
end
```